components/set/SetFormModal.vue

```diff
@@ -50,20 +50,22 @@
           >
             {{ isNew ? $t('actions.goBack') : $t('actions.close') }}
           </b-button>
-          <b-button
-            v-if="!isNew"
-            variant="danger"
-            data-qa="delete button"
-            @click="clickDelete"
-          >
-            {{ $t('set.actions.delete') }}
-          </b-button>
-          <b-button
-            variant="primary"
-            type="submit"
-          >
-            {{ isNew ? $t('set.actions.create') : $t('set.actions.update') }}
-          </b-button>
+          <div class="right">
```

Comment:
Adding the div is enough. The class "right" currently doesn't do anything, could you remove it? 
The space between the red and blue buttons is slightly larger than the design, this can be fixed by adding a d-flex class on this div. 

In other words, can you replace `<div class="right">` with `<div class="d-flex">`?